src/Walker/Collector.C

```diff
@@ -29,69 +29,13 @@ CkReduction::reducerType PDFMerger;
 using walker::Collector;
 
 void
-Collector::chareOrd( const std::vector< tk::real >& ord )
+Collector::chareOrd( const std::vector< tk::real >& ord,
```

Comment:
Not really because I don't actually understand what is going on.

But again, based on the text, something like sumOrdinaryMomentsAndPDFs()? If length is an issue (and that is pretty heinous) sumOrdMomsAndPDF() or along the same lines. That is much more indicative of what that function does than chareOrd()